.github/workflows/auto_approve_dependency_PRs.yml

```diff
@@ -10,15 +10,17 @@ jobs:
       - name: Find dependency PRs
         id: find_prs
         run: |
-          gh pr list --repo "alteryx/woodwork" --author "machineFL" --base main --state open --search "status:success review:required" --limit 1 --json url > dep_PRs_waiting_approval.json
+          rm -f dep_PRs_waiting_approval.json
+          gh pr list --repo "${{ github.repository }}" --author "machineFL" --base main --state open --search "status:success review:required" --limit 1 --json url > dep_PRs_waiting_approval.json
           echo ::set-output name=dep_pull_request_url::$(cat dep_PRs_waiting_approval.json | grep -Eo "(https)://[a-zA-Z0-9./?=_%:-]*")
         env:
           GITHUB_TOKEN: ${{ secrets.REPO_SCOPED_TOKEN }}
       - name: Approve dependency PRs and enable auto-merge
-        if: startsWith( ${{ steps.find_prs.outputs.dep_pull_request_url }}, "https://github.com/alteryx/woodwork/pull/" )
-        run: | 
-          gh pr review --comment --body "auto approve" ${{ steps.find_prs.outputs.dep_pull_request_url }}
-          gh pr review --approve ${{ steps.find_prs.outputs.dep_pull_request_url }}
-          gh pr merge --auto --squash ${{ steps.find_prs.outputs.dep_pull_request_url }}
+        if: steps.find_prs.outputs.dep_pull_request_url != ''
```

Comment:
> When you use expressions in an if conditional, you may omit the expression syntax (${{ }}) because GitHub automatically evaluates the if conditional as an expression. For more information about if conditionals, see "Workflow syntax for GitHub Actions."